### PR TITLE
feat: add @web3-onboard/okx

### PR DESCRIPTION
**Pull request by vanishcode**
### Description

okx web3 wallet extension & mobile app support

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [x] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

#### Tests with demo app (SDK)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

### CODE REVIEW

**Comment by Adamj1232** [comment]
@vanishcode I am seeing the following build errors : https://github.com/blocknative/web3-onboard/actions/runs/9387915377/job/25851950284
Can oyu take a look when you have a moment?

File: packages/okx/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# @web3-onboard/okx
+
+## Wallet module for connecting OKX Wallet through web3-onboard
+
+OKX Wallet SDK wallet module for connecting to Web3-Onboard. Web3-Onboard makes it simple to connect Ethereum hardware and software wallets to your dapp. Features standardized spec compliant web3 providers for all supported wallets, framework agnostic modern javascript UI with code splitting, CSS customization, multi-chain and multi-account support, reactive wallet state subscriptions and real-time transaction state change notifications.
+
+### Install
+
+**NPM**
```
Comment (Adamj1232) [inline packages/okx/README.md:9]:
```suggestion
**NPM**1
```

File: packages/injected/README.md [hunk 2]
```diff
@@ -61,7 +61,6 @@ const onboard = Onboard({
 - Phantom - _Desktop & Mobile_
 - SafePal - _Desktop & Mobile_
 - Zerion - _Desktop & Mobile_
-- OKX Wallet - _Desktop & Mobile_
```
[the remark this answers is not in the document]
Comment (vanishcode) [reply packages/injected/README.md:64]: ok, we will revert this related change 👌

**Comment by vanishcode** [comment]
> @vanishcode I am seeing the following build errors : https://github.com/blocknative/web3-onboard/actions/runs/9387915377/job/25851950284 Can you take a look when you have a moment?

Of course 👌

**Pull request closed without merging**